homeassistant/components/freebox/alarm_control_panel.py

```diff
@@ -25,16 +24,13 @@
     "alarm1_arming": STATE_ALARM_ARMING,
     "alarm2_arming": STATE_ALARM_ARMING,
     "alarm1_armed": STATE_ALARM_ARMED_AWAY,
-    "alarm2_armed": STATE_ALARM_ARMED_NIGHT,
+    "alarm2_armed": STATE_ALARM_ARMED_HOME,
```

Comment:
@nachonam : So is it the `arm_home` or the `arm_nigth` feature that we are getting ?

Reply:
ARMED_HOME and ARMED_NIGHT are more or less the same feature. You are INTO the home and the alarm is ON

Reply:
idle state is also  STATE_ALARM_DISARMED

FREEBOX_TO_STATUS = {
    "alarm1_arming": STATE_ALARM_ARMING,
    "alarm2_arming": STATE_ALARM_ARMING,
    "alarm1_armed": STATE_ALARM_ARMED_AWAY,
    "alarm2_armed": STATE_ALARM_ARMED_HOME,
    "alarm1_alert_timer": STATE_ALARM_TRIGGERED,
    "alarm2_alert_timer": STATE_ALARM_TRIGGERED,
    "alert": STATE_ALARM_TRIGGERED,
    "idle": STATE_ALARM_DISARMED,
}

Reply:
I understand that arm_home and arm_night are very similar but in the Freebox Home meaning, does it represent an arm him or night ?

Ok for the missing idle state, will update accordingly.

Reply:
In  Freebox Home we have alarm1 ("Alarme principale") and alarm2 ("Alarm secondaire")

Reply:
Can they both be activated in the same time ?

Reply:
No, only one alarm can be active (alarm1 or alarm2  but not both)

Reply:
Ok thanks, will fix tomorrow

Reply:
Let's say `arm_home` then

Reply:
> "idle": STATE_ALARM_DISARMED,

idle status is also  needed

Reply:
Oh, yeah forgot

Reply:
idle state added